hpx/util/detail/pack_traversal_async_impl.hpp

```diff
@@ -179,6 +182,27 @@ namespace util {
         {
             Target* target_;
 
+            explicit static_async_range(Target* target) : target_(target) {}
+
+            static_async_range(static_async_range const& rhs) = default;
+            static_async_range(static_async_range && rhs)
```

Comment:
Probably the custom move constructors and operators aren't needed (and probably it is left from debugging).

Reply:
Those are not really needed, indeed. OTOH, explicitly setting the moved-from object to be invalid might be a good thing in general.